src/components/quick-settings.tsx

```diff
@@ -0,0 +1,52 @@
+import { TranslationBundle } from '@jupyterlab/translation';
+import type { CommandRegistry } from '@lumino/commands';
+import { settingsIcon } from '@jupyterlab/ui-components';
+import * as React from 'react';
+import { Menu } from '@lumino/widgets';
+
+import { CommandIDs } from '../types';
+
+export function QuickSettings(props: {
+  trans: TranslationBundle;
+  commands: CommandRegistry;
+}): React.ReactElement {
+  const { commands } = props;
+
+  const menu = new Menu({ commands: commands });
+  menu.addItem({ command: CommandIDs.showStarred, args: {} });
+  menu.addItem({ command: CommandIDs.searchAllSections, args: {} });
+  menu.addItem({
+    command: 'settingeditor:open',
+    args: { query: 'New Launcher' }
```

Comment:
Currently it shows a generic "Settings Editor":

![image](https://github.com/nebari-dev/jupyterlab-new-launcher/assets/5832902/83e9d140-e4c1-407a-903d-171375af7a80)

This is fine, though alternatively we could change it to "Open settings" by wrapping it in a custom command. Of note, it would be good to align the capitalization one way or another (I do not have a preference but I think other menus use Title Caps).